pennylane/qnode.py

```diff
@@ -824,6 +824,15 @@ def __call__(self, *args, **kwargs):  # pylint: disable=too-many-branches, too-m
                     "The device was switched during the call of the QNode, to avoid this behaviour define"
                     "an interface argument instead of auto."
                 )
+        if self.interface == "jax":
+            import jax  # pylint: disable=import-outside-toplevel
+            import jax.numpy as jnp  # pylint: disable=import-outside-toplevel
+
+            if jax.config.read("jax_enable_x64") is False:
+                # We cast to 64-bit floating points to avoid rounding errors
+                jax.config.update("jax_enable_x64", True)

```

Comment:
I'm not sure if we sould change this global variable.

Reply:
Some users might want to keep this value to `False`...

Reply:
you can set it with a context manager IIRC

Reply:
https://github.com/google/jax/blob/c657449528085f140fef8dcb6dc41f8b136ba0f8/tests/x64_context_test.py#L40

Reply:
Now the config is updated during PL import.